src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -64,12 +85,11 @@ public function formatSummary(
         }
 
         return sprintf(
-            'Request made: Host: "%s", Method: "%s", ClientCustomerId: %d, RequestId: "%s", '
+            'Request made: Host: "%s", Method: "%s", CustomerId: %s, RequestId: "%s", '
```

Comment:
You can keep `%d` if you decide to return an int (see previous comment)

Reply:
No. See my reply above.